featurebyte/schema/credential.py

```diff
@@ -94,6 +97,7 @@ class CredentialServiceUpdate(BaseDocumentServiceUpdateSchema):
     Credential service update schema
     """
 
+    group_ids: Optional[List[PydanticObjectId]] = Field(default=None)
```

Comment:
Do we need to validate the the group ids are valid (check that the group actually exists)?

Reply:
the credential api+controller will be overridden in featurebyte-app